extensions/ipynb/package.json

```diff
@@ -22,8 +22,8 @@
     "workspace",
     "ui"
   ],
-  "main": "./out/ipynbMain.js",
-  "browser": "./dist/browser/ipynbMain.js",
+  "main": "./out/ipynbMain.node.js",
+  "browser": "./dist/browser/ipynbMain.browser.js",
```

Comment:
This way we have a full proof way to determine whether the runtime context is browser or node.js
See `ipynbMain.ts` file. (passing extra argument `isBrowser`)